css/app.css

```diff
@@ -525,7 +539,18 @@ header.hide-controls #window-close {
   overflow: hidden;
 }
 
-#editor-textarea:focus {
+#editor-textarea:focus, #editor-textarea:hover {
+  outline: none;
+}
+
+
+.editor-textarea-container.dark-theme .editor-textarea-wrapper {
+  background: var(--dark-background-color);
+  color: var(--dark-theme-text-color);
+}
+
+#ed
```

Comment:
i'm guessing this newline is a mistake

Reply:
yep haha, interesting that it didn't effect behavior